Library/Homebrew/utils/github.rb

```diff
@@ -245,7 +245,7 @@ def issues_for_formula(name, options = {})
   end
 
   def print_pull_requests_matching(query)
-    open_or_closed_prs = search_issues(query, type: "pr")
+    open_or_closed_prs = search_issues(query, type: "pr", user: "Homebrew")
```

Comment:
An potential alternative would be to use `, org: "Homebrew"` here instead.

Reply:
Let me know if you feel strongly about this & I'll follow-up.